pkgs/by-name/xr/xrsh/package.nix

```
@@ -0,0 +1,65 @@
+{
+  stdenv,
+  fetchFromGitea,
+  makeWrapper,
+  writeShellApplication,
+  python3Packages,
+  lib,
+  unstableGitUpdater,
```

Alphabetical and logical ordering.

```suggestion
  lib,
  stdenv,
  fetchFromGitea,
  makeWrapper,
  unstableGitUpdater,
  writeShellApplication,

  python3Packages,
```

---

done 
(is this actually part of the contributors guidelines?)